pkg/virtctl/vm/vm.go

```diff
@@ -511,6 +511,148 @@ func expandVirtualMachine(namespace string, virtClient kubecli.KubevirtClient, o
 	return nil
 }
 
+func startVirtualMachine(vmiName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
+	err := virtClient.VirtualMachine(namespace).Start(context.Background(), vmiName, &v1.StartOptions{Paused: startPaused, DryRun: *dryRunOption})
+	if err != nil {
+		return fmt.Errorf("Error starting VirtualMachine %v", err)
+	}
+
+	return nil
+}
+
+func stopVirtualMachine(vmiName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
+	if gracePeriodIsSet(gracePeriod) && forceRestart == false {
```

Comment:
We have multiple if statements that check `gracePeriodIsSet` can we reduce and simplify them into a single one, maybe avoiding too many nested if statements

Reply:
I've made changes to the stopVM and restartVM functions to simplify the checks of gracePeriodIsSet. Please let me know what you think.